### PR TITLE
Display AzurePowershell Survey url for Resolve-Error

### PR DESCRIPTION
**Pull request by msJinLei**
issue : https://github.com/Azure/azure-powershell/issues/11021

## Checklist

- [ ] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ ] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/Accounts/Accounts.Test/ErrorResolutionTests.cs [hunk 1]
```diff
@@ -29,6 +31,56 @@ namespace Microsoft.Azure.Commands.Profile.Test
...
+                InformationRecord informationRecord = new InformationRecord(messageData, String.Empty);
+
+                if (tags != null)
+                {
+                    informationRecord.Tags.AddRange(tags);
+                }
+                commandRuntime2.WriteInformation(informationRecord);
+            }
+
+            public override void ExecuteCmdlet()
```
Comment (dingmeng-xue) [inline src/Accounts/Accounts.Test/ErrorResolutionTests.cs]: I'd like to the reason why ExecuteCmdlet() method needs to be overridden.

Comment (msJinLei) [reply src/Accounts/Accounts.Test/ErrorResolutionTests.cs]: > I'd like to the reason why ExecuteCmdlet() method needs to be overridden.

Change the implementation by using a virtual method wrapping the WriteInformation.

**Pull request closed without merging**